ecs_logging/_stdlib.py

```diff
@@ -42,9 +43,24 @@ class StdlibFormatter(logging.Formatter):
     }
     converter = time.gmtime
 
-    def __init__(self):
-        # type: () -> None
+    def __init__(self, include_exc_info=False, stack_trace_limit=0):
```

Comment:
Is there any prior-art for the names of parameters on Formatters for "include_exc_info"?
Also feels strange doing one using Python terminology (`exc_info`) and one using ECS terminology (`stack_trace`), might be good to only use ECS terminology?